packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -42,10 +42,10 @@ Requires: %{?scl_prefix}rubygem(apipie-rails) >= 0.5.14
 Requires: %{?scl_prefix}rubygem(runcible) >= 2.11.0
 Requires: %{?scl_prefix}rubygem(runcible) < 3.0.0
 Requires: %{?scl_prefix}rubygem(anemone)
-Requires: %{?scl_prefix}rubygem(pulpcore_client)
-Requires: %{?scl_prefix}rubygem(pulp_file_client) < 3.0.0
-Requires: %{?scl_prefix}rubygem(pulp_ansible_client)
-Requires: %{?scl_prefix}rubygem(pulp_docker_client)
+Requires: %{?scl_prefix}rubygem(pulpcore_client) <= 3.0.0rc5.dev01566405557
+Requires: %{?scl_prefix}rubygem(pulp_file_client) <= 0.1.0b2.dev01566325827
+Requires: %{?scl_prefix}rubygem(pulp_ansible_client) <= 0.2.0b3.dev01566484022
+Requires: %{?scl_prefix}rubygem(pulp_docker_client) <= 4.0.0b6.dev01566480729
```

Comment:
didn't we have issues with adding such specific limits to pulp client in the past? @jlsherrill